### PR TITLE
docs: fix broken links

### PR DESCRIPTION
**Pull request by lukapetrovic**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Links are broken.

Issue Number: N/A

## What is the new behavior?

Working links.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/npm-packages.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ The following support packages are included as dependencies in the default `pack
...
-[**zone.js**](https://github.com/angular/zone.js) | Angular relies on zone.js to run Angular's change detection processes when native JavaScript operations raise events. Zone.js is an implementation of a [specification](https://gist.github.com/mhevery/63fdcdf7c65886051d55) currently before the [TC39](http://www.ecma-international.org/memento/TC39.htm) committee that determines standards for the JavaScript language.
+[**zone.js**](https://github.com/angular/zone.js) | Angular relies on zone.js to run Angular's change detection processes when native JavaScript operations raise events. Zone.js is an implementation of a [specification](https://gist.github.com/mhevery/63fdcdf7c65886051d55) currently before the [TC39](https://www.ecma-international.org/memento/tc39-m.htm) committee that determines standards for the JavaScript language.
```
Comment (gkalpak) [inline aio/content/guide/npm-packages.md]: Why not `https://www.ecma-international.org/memento/tc39.htm` (without the `-m` before `.htm`)?

**Review by gkalpak — APPROVED** [review]
Thx, @lukapetrovic :+1:
Just a minor comment. Otherwise lgtm!

File: aio/content/guide/npm-packages.md (hunk 1 above)
Comment (lukapetrovic) [reply aio/content/guide/npm-packages.md]: https://www.ecma-international.org/memento/tc39-m.htm is used in the package name just before this one. But I can change both to point to the committee homepage https://www.ecma-international.org/memento/tc39.htm.

Comment (gkalpak) [reply aio/content/guide/npm-packages.md]: Good catch! Changing both sounds good to me, thx :+1:

**Review by gkalpak — APPROVED** [review]
Thx for making the changes, @lukapetrovic!

**Comment by gkalpak** [comment]
**merge-assistance:** Global approval for this docs-only change.

**Pull request closed without merging**